c_emulator/riscv_platform.c

```diff
@@ -37,6 +37,11 @@ bool sys_enable_zfinx(unit u)
   return rv_enable_zfinx;
 }
 
+bool sys_enable_fiom(unit u)
+{
+  return rv_enable_fiom;
```

Comment:
Agreed, that is a much better name! I've updated it everywhere.

Reply:
I don't see the changes.  What am I missing?

Reply:
I think it's just because this comment stays linked to the first commit and gets marked "Outdated". I added a second commit with the rename. If you go to the "Files changed" tab at the top you should see it.